blaze-client/src/main/scala/org/http4s/client/blaze/ClientTimeoutStage.scala

```diff
@@ -32,12 +32,12 @@ final private class ClientTimeoutStage(idleTimeout: Duration, requestTimeout: Du
   override def name: String = s"ClientTimeoutStage: Idle: $idleTimeout, Request: $requestTimeout"
 
   /////////// Private impl bits //////////////////////////////////////////
-  private val killswitch = new Runnable {
+  private def killswitch(name: String, timeout: Duration) = new Runnable {
     override def run(): Unit = {
-      logger.debug(s"Client stage is disconnecting due to timeout.")
+      logger.debug(s"Client $name stage is disconnecting due to timeout after $timeout.")
```

Comment:
Hmm would this make more sense if I made it `logger.debug(s"Client stage is disconnecting due to $name timeout after $timeout.")`?